utils/infer_utils.py

```diff
@@ -21,11 +21,12 @@
 
 from utils.misc_utils import get_center
 
+"""Notice in Rectangle namedtuple object, x and y can be top-left or center"""
```

Comment:
Replace the comments with
Note that x and y can be the top-left or center coordinates of the rectangle